src/IssueViz.Security.UnitTests/Taint/TaintList/TaintToolWindowTests.cs

```diff
@@ -44,7 +57,25 @@ public void Dispose_FrameIsClosed()
 
         private TaintToolWindow CreateTestSubject(Mock<IVsWindowFrame> frameMock)
         {
+            var componentModelMock = new Mock<IComponentModel>();
```

Comment:
Added services required to create the view model; same as the equivalent hotspot tests.

This files is now almost identical to the hotspot tool window tests, but they each only have one test (of Dispose). I'll have a look at this later as part of the refactoring to see if it's worth removing the duplicated product/test code.

Reply:
I'm not sure that the Dispose code is needed.

I originally copied it from the secondary locations code (IssueVisualizationToolWindow), where it was needed due to a weird integration with ProvideToolWindowVisibility.
I thought it would be good for the 2 new windows as well, but it wasn't, and I later added `ProvideToolWindow(Transient = true)` which worked for them.
But I didn't remove this code and I think it was a mistake. `Transient = true` should be enough if we're not using ProvideToolWindowVisibility.
I'll open a ticket to investigate it and if it's not needed, we could remove these tests entirely.

Reply:
Ok. It would have be worth commenting in the Dispose method why it was there.